src/main/resources/templates/admin/moduleView.html

```diff
@@ -0,0 +1,535 @@
+<!DOCTYPE HTML>
+<html xmlns:th="http://www.thymeleaf.org">
+<head th:replace="~{/admin/header :: head}"/>
+<body>
+
+<span th:replace="~{/admin/header :: top}"/>
+
+<a href="/admin/">Home</a>
+-
+<a href="/admin/modules">Modules</a>
+
+<th:block th:if="${canManageModules}">
+    <h2 th:text="|Module ${module.name}|">Module (name)</h2>
+
+    <p>
+        Preview module content and schedule events. Event schedules are all in UTC. CMD or CRTL +
+        click to select multiple projects for adding/removing from events
+    </p>
+
+    <h3> Cohorts and Projects </h3>
+    <div th:each="cohort : ${module.cohorts}">
+        <h4 th:text="${cohortNames.get(cohort.cohortId)} + ' '
+            + ${#temporals.format(cohort.startDate, 'yyyy/MM/dd')} + ' - '
+            + ${#temporals.format(cohort.endDate, 'yyyy/MM/dd')}">
+            Cohort Name (Start Date - End Date)
+        </h4>
+        <ul th:each="projectId : ${cohort.projects}">
+            <li th:text="${projectNames.get(projectId)}"> Project Name </li>
+        </ul>
+    </div>
+
+
+    <th:block th:if="${module.overview}">
+        <hr>
+        <h3> Overview </h3>
+        <div data-th-utext="${module.overview}"> </div>
```

Comment:
Use `th:utext` instead of `data-th-utext` for consistency with the other Thymeleaf attributes; this file isn't valid HTML5 anyway.